### ISSUE TITLE
Update functional graphics / playstore screenshots

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#11915]
New icons and UI changes should be visible in our functional graphic and in playstore screenshots.

---

This file will help while updating the graphic:

https://github.com/cgeo/cgeo/blob/master/main/project/playstore/screenshots/aditional%20data/cgeo%20playstore%20images%20-%20functional%20graphic.pptx

_Originally posted by @fm-sys in https://github.com/cgeo/cgeo/discussions/11913#discussioncomment-1486665_

**Comment by fm-sys** [comment]
And this one for the screenshots:
https://github.com/cgeo/cgeo/blob/master/main/project/playstore/screenshots/aditional%20data/cgeo%20playstore%20images%20-%20screenshots.pptx

**Comment by moving-bits** [comment]
I propose to wait with updating the graphics until we got feedback using the icon survey, and have incorporated any changes (if needed), to not having to update the graphics twice.

**Comment by moving-bits** [comment]
> Why should we present the old icons in the survey?

I meant the screenshots for play store, the survey should present the new icons

**Comment by ztNFny** [comment]
![bild](https://github.com/cgeo/cgeo/blob/653dfa7f7b0e233b70daa08f163bb4a123019e42/main/project/playstore/icons/cgeo_functional_graphic.jpg?raw=true)

_Originally posted by @fm-sys in https://github.com/cgeo/cgeo/issues/12466#issuecomment-1004401348_

***

Lego in London? ;-)

**Comment by fm-sys** [comment]
> Lego in London? ;-)

The map is actually showing Berlin ;-)

**Comment by ztNFny** [comment]
Potsdamer 0lstz lokks very london-like...

**Comment by Lineflyer** [comment]
Friendly reminder if someone ( @ztNFny  @fm-sys ??) would like to work in it.

Now that we have the new icons as well as an more-or-less final bottom navigation we should update the styling for those screenshots and social media title imgage as well.

**Comment by fm-sys** [comment]
I would like to help and also still have all files from the last time when we created the screenshots at hand. Unfortunately I will probably not have the necessary time in the next two months. 
If you are OK with this delay, I would be happy to work on the screenshots a second time.

To be honest, I don't think that users care that much about the screenshots.

**Comment by fm-sys** [comment]
Nearly everything has changed since I created the old new screenshots in 2020 as on of my very first contributions.

I could think about creating new ones, however I would like to discuss first how they should look like to save me some time.

- Is the general design still considered modern?

- Is the diagonal phone on the first two screenshots still wanted?

- do you want to show any new features which where introduced since then?

**Comment by eddiemuc** [comment]
I am fine with the current design.

New features: I would love it when we can introduce the Wherigo Player of course :-). Also offline translations very quickly turned into a "never-want-to-live-without-"feature for me, but that one is probably not so easy to "visualize".

With regards to design: since we are currently having some glitches due to SDK 35 (eg https://github.com/cgeo/cgeo/issues/17151) it is maybe better to wait until those are resolved before doing new screenshots?

**Comment by ztNFny** [comment]
> ![bild](https://github.com/cgeo/cgeo/blob/653dfa7f7b0e233b70daa08f163bb4a123019e42/main/project/playstore/icons/cgeo_functional_graphic.jpg?raw=true)
> 
> _Originally posted by @fm-sys in https://github.com/cgeo/cgeo/issues/12466#issuecomment-1004401348_
> 
> ***
> 
> Lego in London? ;-)

For an image like this I'd go for something more akin to the event banner - large orange background with the roads

**Comment by moving-bits** [comment]
> I am fine with the current design.

me as well

> With regards to design: since we are currently having some glitches due to SDK 35 (eg [#17151](https://github.com/cgeo/cgeo/issues/17151)) it is maybe better to wait until those are resolved before doing new screenshots?

or use current release version (is not yet targeted against SDK 35) or use a mobile/emulator not running Andrid 15+

**Comment by fm-sys** [comment]
> For an image like this I'd go for something more akin to the event banner - large orange background with the roads

Fine for me! Do you want to take care of the functional graphic and I'll do the screenshots? 
Although I'm still really in love with the foggy forest :)

**Comment by Lineflyer** [comment]
> Although I'm still really in love with the foggy forest :)

I also like the foggy forest as it is somehow a contrast to the rest of the styling colors and brings a bit of "outdoor feeling".

**Comment by JakeDot** [comment]
Is this ticket still needed?

**Comment by fm-sys** [comment]
> Is this ticket still needed?

Yes, the screenshots are quite old again. Unfortunately I'm not able to work on it myself due to lack of time.
